src/configuration/utils.ts

```diff
@@ -9,8 +9,12 @@ export async function getValidHostname (hostname: string): Promise<string> {
         if (!valid)
             throw new GeneralError(RUNTIME_ERRORS.invalidHostname, hostname);
     }
-    else
-        hostname = endpointUtils.getIPAddress();
+    else {
+        const ip = endpointUtils.getIPAddress();
+
+        if (ip)
+            hostname = ip;
+    }
 
     return hostname;
```

Comment:
If the getIPAddress() method returns unidentified, should we handle this error here before proceeding?

@Aleksey28, What do you think about this?